noxfile.py

```diff
@@ -49,7 +49,6 @@ def set_iree_state():
     "IREE_INDEX_URL": os.getenv(
         "IREE_INDEX_URL", "https://iree.dev/pip-release-links.html"
     ),
-    "PYBAMM_DISABLE_TELEMETRY": "true",
```

Comment:
As far as I could tell, this was not working. It seemed like telemetry was still running in tests.